src/Ocelot/DependencyInjection/IOcelotBuilder.cs

```diff
@@ -4,11 +4,15 @@
 using System.Net.Http;
 using IdentityServer4.AccessTokenValidation;
 using Ocelot.Middleware.Multiplexer;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
 
 namespace Ocelot.DependencyInjection
 {
     public interface IOcelotBuilder
     {
+        IServiceCollection Services { get; }
```

Comment:
Why do we need to do this? People can already edit these outside of Ocelot builder if they want to.

Reply:
Ok I understand